proxy/api/src/http/project.rs

```diff
@@ -49,13 +50,36 @@ fn get_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Re
         .and_then(handler::get)
 }
 
-/// `GET /`
-fn list_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
-    http::with_context(ctx)
+/// `GET /tracked`
+fn tracked_filter(
+    ctx: context::Ctx,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    path("tracked")
+        .and(warp::get())
+        .and(http::with_context(ctx))
+        .and(path::end())
+        .and_then(handler::list_tracked)
+}
+
+/// `GET /contributed`
+fn contributed_filter(
+    ctx: context::Ctx,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    path("contributed")
+        .and(warp::get())
+        .and(http::with_context(ctx))
+        .and(path::end())
+        .and_then(handler::list_contributed)
+}
+
+/// `GET /user/<id>
+fn peer_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
```

Comment:
```suggestion
fn tracked_by_user_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
```

Reply:
Again, these aren't necessarily "tracked" projects as per our definition of tracked

Reply:
hm that's not what the docs of `pub async fn list_for_user` nor `pub fn list_projects_for_user` suggest, neither the implementation of `pub fn list_projects_for_user`. 

is this a matter of language? :de: :netherlands: :england:

Reply:
Ya, I'm looking at them now and the docs are wrong :smile: 

I'm going to update them and the function name now :+1: Thanks for calling it out

Reply:
Ah great :ok_hand: